server/config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
   gcloudDatastoreNamespace: 'living-meta-analysis-v2',
   port: 8080,
   // comment out, or delete, the following lines, to have LiMA running on HTTP only
-  httpsPort: 8443,
-  httpsCert: '/etc/letsencrypt/live/lima.soc.port.ac.uk/fullchain.pem',
-  httpsKey: '/etc/letsencrypt/live/lima.soc.port.ac.uk/privkey.pem',
+  // httpsPort: 8443,
```

Comment:
please remove this from the PR

FYI you can run lima with `DISABLE_HTTPS=1 npm run forever` and you won't need to comment this block out